pkg/gatherers/clusterconfig/openshift_storage_test.go

```diff
@@ -0,0 +1,84 @@
+package clusterconfig
```

Comment:
Same here with the `//nolint: dupl`

Reply:
should be fixed by https://github.com/openshift/insights-operator/pull/614/commits/0e6c12b3c6de3c125dc1198bd897092c3570acb7